netutils/ip.py

```diff
@@ -60,12 +60,13 @@ def ipaddress_interface(ip: str, attr: str) -> t.Any:
     return retrieved_method
 
 
-def ipaddress_network(ip: str, attr: str) -> t.Any:
+def ipaddress_network(ip: str, attr: str, **kwargs: t.Any) -> t.Any:
     """Convenience function primarily built to expose ipaddress.ip_network to Jinja.
 
     Args:
         ip: IP network str compliant with ipaddress.ip_network inputs.
         attr: An attribute in string dotted format.
+        kwargs: If called a method allows for passing kwargs to resulting method call.
```

Comment:
```suggestion
        kwargs: Keyword arguments where the key is the method and the value is the value to send to the given method of ipaddress.ip_network.
```

Reply:
so `attr` still remains the method.  kwargs are literally kwargs we're directly passing to the method that `attr` is specifying.

Reply:
you good leaving this one as is @itdependsnetworks ?